gulpfile.babel.js

```diff
@@ -69,13 +72,13 @@ function lint(files, options) {
 gulp.task('lint:client', lint(`${clientDir}/**/*.jsx`));
 gulp.task('lint:server', lint(`./${serverDir}/server.js`));
 gulp.task('lint:step-templates', () => {
-  return gulp.src('./step-templates/*')
+  return gulp.src('./step-templates/*.json')
```

Comment:
We do `*` here as sometimes customers fail to save their step template as a `.json` file and this is caught here